Src/Tracer.Logger.EurekaLog.pas

```diff
@@ -44,6 +44,17 @@ procedure TEurekaLogRefCountTracerLog.DoLogStackTrace;
 
 procedure TEurekaLogRefCountTracerLog.LogLine(
   const CallStack: TEurekaBaseStackList; const Index: Integer);
+
+  function GetAddressOf(var AVariable): string;
+  begin
+    Result := IntToHex( Integer( Pointer( @AVariable ) ), 8 );
+  end;
+
+  function PointerToHex(var AVariable): string;
```

Comment:
Uups, if that unit wasn't included it didn't even compile!? Good catch. Now it's ready to merge.